app/models/loader/types.rb

```diff
@@ -96,43 +96,58 @@ def create_resource!
           resource.annotations_dataset.import(%i(resource_id name value), records)
         end
       end
-      
+
       def resource
         ::Resource[resourceid]
       end
     end
-      
+
     class Record < Types::Base
       include CreateRole
       include CreateResource
-      
+
+      def verify
```

Comment:
Can we use a more descriptive method name? `validate_record_values` maybe?

Reply:
its not only values - it can be also match with other policy parts

Reply:
Could we capture this interface somewhere else? It feels like it's just adding a bunch of code without any value. Can we remove the interface stubbing until we actually implement the feature?

Reply:
The implementation is going to be done in the shortest future by Nessi